tox.ini

```diff
@@ -4,14 +4,12 @@
 
 [tox]
 envlist =
-        py27-django{1.11},
-        py36-django{1.11,2.2,3.0}
+        py36-django{2.2,3.0}
```

Comment:
Also would be nice to see here Python3.7

Reply:
There's a plenty more python versions we can test against but given the scope of this project I don't see much value